src/main/java/com/qovery/intellij/plugin/completion/value/ContextualDatabaseVersion.java

```diff
@@ -8,8 +8,9 @@
 
 public class ContextualDatabaseVersion implements ContextualValueSet {
 
-    private final String[] POSTGRES_VERSION = {"9.4", "9.5", "9.6", "10", "11"};
+    private final String[] POSTGRES_VERSION = {"9", "10", "11", "12"};
     private final String[] MYSQL_VERSION = {"5.5", "5.6", "5.7", "8.0"};
+    private final String[] MONGO_VERSION = {"3.6", "4.0", "4.2"};
```

Comment:
Can you also add 4.4 please

Reply:
Just saying, the documentation does not show the 4.4 version (https://docs.qovery.com/docs/using-qovery/configuration/databases/mongodb)